recipes/yarn.rb

```diff
@@ -8,6 +8,11 @@
 home = node['hops']['hdfs']['user_home']
 private_ip=my_private_ip()
 
+if node['install']['localhost'].casecmp?("true")
+   node.override['hadoop_spark']['hopsutil']['url'] = "#{node['download_url']}/hopsutil/#{node['install']['version']}-insecure/hops-util-#{node['install']['version']}.jar"
```

Comment:
This has 2 issues: 
- We do not assemble any download link in the recipes. We do that in the `attributes`. If you assemble the link here it breaks `setup-chef`.
- I already pointed out that having 2 builds one for secure and one for insecure is not sustainable. I guess you copied on snurran the artifact from 0.10.0-insecure to 1.0.0-insecure to make it work. Problem is that as soon as someone pushes something on `hops-util` this PR will stop working as it will download an old version.

Please talk to @tkakantousis. Either add a profile to maven so that Jenkins can build both versions or add a variable to be passed to the spark app.